app/build.gradle

```diff
@@ -141,6 +141,7 @@ dependencies {
     // KotlinX Serialization
     implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:$serialization_version"
     implementation "com.jakewharton.retrofit:retrofit2-kotlinx-serialization-converter:0.8.0"
+    implementation "org.jetbrains.kotlinx:kotlinx-datetime:0.4.0"
```

Comment:
Any reason for not choosing the Java DateTime API which offers quite more functionality?

Reply:
I dont know the difference actually!. I will look at the Java DateTime Api.

Reply:
I have implemented the Java DateTime Api, it seems like a drop-in replacement.